py3status/modules/mpd_status.py

```diff
@@ -110,12 +110,30 @@ class Py3status:
     state_play = '[play]'
     state_stop = '[stop]'
 
+    def _connection(self):
```

Comment:
Rename `self._connection` to `self._get_mpd` to remove the confusion between `self.connection` and `self._connection`.

Reply:
that was intentional, but I'll change it.

Reply:
`_get_mpd` feels off to me. how about `_get_connection()`?

Reply:
```
    playlist_info = self._get_connection().playlistinfo()
    status = self._get_connection().status()
    self._get_connection(disconnect=True)
```
versus...
```
    playlist_info = self._get_mpd().playlistinfo()
    status = self._get_mpd().status()
    self._get_mpd(disconnect=True)
```

Reply:
yeah, i see. (not my style, but ok :P)